themes/frontierline/inc/customizer.php

```diff
@@ -94,6 +94,22 @@ function frontierline_customize_register($wp_customize) {
     'type'        => 'checkbox',
   ));
 
+  // Register featured image aka post thumbnail option and control
+  $wp_customize->add_setting('frontierline_no_byline', array(
+    'capability'        => 'edit_theme_options',
+    'default'           => '',
```

Comment:
Can you make the default value so, that in the parent theme the byline is displayed, and in the child not, so the behaviour does not change when updating the theme? I think that should be possible in the respective `functions.php` for each theme.

Reply:
This is a good idea and I found references to people doing this but no code examples. I was not able to get it working myself :(

Reply:
I tried a few things too and ran into similar problems. Using `$wp_customize->get_setting()` seems to have no effect, and using `set_theme_mod()` is effective but then doesn't seem reversible. This seems like it should be doable but actually doing it proves more troublesome... I don't know the customizer API well enough to know what we're missing. Maybe @MikkCZ can have better success?

Since this PR is blocking us switching over a bunch of press blogs to Frontierline I think we should move forward with it and just set the option on the Firefox blog manually. We can revisit the issue another time.

Reply:
I do not have a solution in my head right now. Also I will be afk until the next week and I do not want to block you. In the end it's only changing a preference on a single blog.